src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-delete.test.ts

```diff
@@ -69,18 +82,18 @@ describe("DELETE /jira/connect/enterprise/app/:uuid", () => {
 	});
 
 	it("should send a successful response when app is deleted", async () => {
-		const response = mockResponse();
-		await JiraConnectEnterpriseAppDelete(mockRequest("95980446-16e1-11ed-861d-0242ac120002"), response, jest.fn());
```

Comment:
Tbh, I have no idea where does this `95980446-16e1-11ed-861d-0242ac120002` comes from.